configuration.md

```diff
@@ -0,0 +1,61 @@
+# Policy Controller Configuration 
```

Comment:
Calico Policy Controller Configuration?

At first I was wondering why we were documenting k8s policy controller, but then realized it is ours